src/Tribe/Aggregator/Records.php

```diff
@@ -147,7 +147,7 @@ public function get_post_type() {
 
 		$args = array(
 			'description'        => esc_html__( 'Events Aggregator Record', 'the-events-calendar' ),
-			'public'             => true,
+			'public'             => false,
```

Comment:
We set it to `true` but overrode `show_in_nav_menus`, `show_in_ui` and `publicly_queryable` setting them to `false`.
This left `exclude_from_search` set to `false` (opposite of `public`), from the Codex:

`exclude_from_search`

(boolean) - Whether to exclude posts with this post type from front end search results.
Default: value of the opposite of public argument

* 'true' - site/?s=search-term will not include posts of this post type.
* 'false' - site/?s=search-term will include posts of this post type.

Note: If you want to show the posts's list that are associated to taxonomy's terms, you must set exclude_from_search to false (ie : for call site_domaine/?taxonomy_slug=term_slug or site_domaine/taxonomy_slug/term_slug). If you set to true, on the taxonomy page (ex: taxonomy.php) WordPress will not find your posts and/or pagination will make 404 error...

Since the only value we left of the `public=true` setting would deal with the front-end this seems a safe change, anything particular you are thinking of?

Reply:
I can't really remember why that was the case. But lets go with it, can you highlight that on the ticket for QA?